content/05_agreg.md

```diff
@@ -24,11 +24,346 @@ Cette section traite de :
 
   Le fichier que vous devez modifier pour ce chapitre est `mongo_book/content/05_agreg.md`.
 
-## Successions d'étapes d'agrégation
+## <center> Successions d'étapes d'agrégation </center>
```

Comment:
Merci de ne pas toucher à la mise en forme des titres pour conserver l'homogénéité du document